logstash-core/lib/logstash/runner.rb

```diff
@@ -79,6 +79,15 @@ class LogStash::Runner < Clamp::Command
     I18n.t("logstash.runner.flag.node_name"),
     :attribute_name => :node_name
 
+  def validate_positive_integer(str_arg)
```

Comment:
I think you're mostly right, but its a little premature. I'd agree if we had two validations, but since its only one, I'd rather not create a one method utility class.